CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ This project adheres to [Semantic Versioning](https://semver.org/).
   (#[1456](https://github.com/nix-rust/nix/pull/1456))
 - Added `TcpUserTimeout` socket option (sockopt) on Linux and Fuchsia.
   (#[1457](https://github.com/nix-rust/nix/pull/1457))
+  (#[1447](https://github.com/nix-rust/nix/pull/1447))
```

Comment:
Somehow you duped a line here.
```suggestion
```

Reply:
Fixed (I don't know where it comes from! probably something during rebase with master) Thanks